src/utils/auth.js

```diff
@@ -131,3 +117,23 @@ module.exports = {
   getCachedToken,
   getLoggedUser
 }
+
+/****** Utilities ******/
+function getSafeUser(user) {
+  return omit(user, ['__v', '_id', 'password'])
+}
+
+function signUserToken(id) {
+  return jwt.sign({ id }, privateKey, { expiresIn: '2h' })
+}
+
+function getCookieOpts() {
+  const afterThirtyDays = new Date(Date.now() + 30 * 24 * 60 * 60)
+
+  const cookieOptions = {
+    httpOnly: true,
+    expires: afterThirtyDays,
+    signed: true
+  }
+  return cookieOptions
+}
```

Comment:
What's the point of setting cookie expiration to 30 days while jwt token expires after 2h?

Reply:
Great catch. That's a valid question and needs to be changed to synchronize the cookie with the token.

Besides that, there's a prevalent practice using refresh tokens but I didn't look into that tbh. It's a possible improvement to implement.

Reply:
For now, it doesnt matter anyway. A new cookie will generated when jwt is outdated. Right?

Reply:
indeed, since the user will have to login manually. But I went ahead and synchronized them anyway

Reply:
Great!